common/customizable_localization/00_ruler_transition_loc.txt

```diff
@@ -367,6 +367,16 @@ DeathMessageOutro = {
 		}
 	}
 
+	# Warcraft
+	text = {
+		trigger = { has_trait = magic_good }
+		localization_key = SUCCESSION_EVENT_WINDOW_DEAD_OUTRO_WARCRAFT_MAGIC_GOOD
+	}
+	text = {
+		trigger = { has_trait = magic_bad }
+		localization_key = SUCCESSION_EVENT_WINDOW_DEAD_OUTRO_WARCRAFT_MAGIC_BAD
+	}
+
```

Comment:
In vanilla, non-inheritable traits are included here.

Reply:
> In vanilla, non-inheritable traits are included here.

Can't confirm?

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/105057346-2ce11c80-5a86-11eb-9fad-6064a715a563.png)

</details>

Reply:
Nvm.